package.json

```diff
@@ -40,7 +40,7 @@
     "hellojs": "1.14.1",
     "local-storage": "1.4.2",
     "lodash": "4.17.4",
-    "loglevel": "1.4.1",
+    "loglevel": "github:thomasconner/loglevel",
```

Comment:
@thomasconner what were the changes you made to the package? Do you plan to keep your fork updated periodically from the master?